features/eif-adapter/src/main/java/org/opennms/features/eifadapter/EifMessageDecoder.java

```diff
@@ -0,0 +1,54 @@
+package org.opennms.features.eifadapter;
```

Comment:
Missing copyright header. See https://github.com/OpenNMS/opennms/blob/develop/core/ipc/rpc/api/src/main/java/org/opennms/core/rpc/api/RpcClient.java#L1 for an example.